tests/tool/Integration.Tests/E2ETest.cs

```diff
@@ -34,27 +41,23 @@ public async Task UpgradeTest(string scenarioName, string language, string input
         {
             // Create a temporary working directory
             var workingDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
-            try
-            {
-                var dir = Directory.CreateDirectory(workingDir);
-                Assert.True(dir.Exists);
+            var dir = Directory.CreateDirectory(workingDir);
+            Assert.True(dir.Exists);
 
-                // Copy the scenario files to the temporary directory
-                var scenarioDir = Path.Combine(IntegrationTestAssetsPath, scenarioName, language);
-                await CopyDirectoryAsync(Path.Combine(scenarioDir, OriginalProjectSubDir), workingDir).ConfigureAwait(false);
+            // Copy the scenario files to the temporary directory
+            var scenarioDir = Path.Combine(IntegrationTestAssetsPath, scenarioName, language);
+            await CopyDirectoryAsync(Path.Combine(scenarioDir, OriginalProjectSubDir), workingDir).ConfigureAwait(false);
 
-                // Run upgrade
-                await UpgradeRunner.UpgradeAsync(Path.Combine(workingDir, inputFileName), entrypoint, Console.Out, 300).ConfigureAwait(false);
-                CleanupBuildArtifacts(workingDir);
+            // Run upgrade
+            await UpgradeRunner.UpgradeAsync(Path.Combine(workingDir, inputFileName), entrypoint, _output, 300).ConfigureAwait(false);
 
-                await AssertDirectoriesEqualAsync(Path.Combine(scenarioDir, UpgradedProjectSubDir), workingDir).ConfigureAwait(false);
-            }
-            finally
```

Comment:
Was the try/finally just removed because UpgradeAsync won't throw or is there another reason?

Reply:
I didn't want the test directories deleted if it failed. This way you can go to the directory and see what failed